tests/fetch/to_indexeddb.cpp

```diff
@@ -50,4 +46,5 @@ int main()
   emscripten_fetch_t *fetch = emscripten_fetch(&attr, "gears.png");
   assert(fetch != 0);
   memset(&attr, 0, sizeof(attr)); // emscripten_fetch() must be able to operate without referencing to this structure after the call.
+  return 99;
```

Comment:
There is an example of where we actually do return 99 and we expect it not to be reported as an exit (since there is outstanding async work).  If we ever see "failed: 99" will can guess what went wrong.